src/JsConverter.php

```diff
@@ -22,18 +22,26 @@ public static function convertToJson(string $jsObjectString)
     {
         $replacedStringsList = [];
 
-        // 1. Replace all delimited string literals with placeholders
-        $convertedString = self::replaceSectionsWithPlaceholders($jsObjectString, $replacedStringsList, "'");
+        $convertedString = $jsObjectString;
+
+        // 0. Remove functions from objects
```

Comment:
Super minor change to this comment (I can't count apparently)

```suggestion
        // 1. Remove functions from objects
```